trackpy/feature.py

```diff
@@ -284,28 +248,10 @@ def refine(raw_image, image, radius, coords, separation=0, max_iterations=10,
     # Flat peaks return multiple nearby maxima. Eliminate duplicates.
     if np.all(np.greater(separation, 0)):
         mass_index = image.ndim  # i.e., index of the 'mass' column
-        while True:
```

Comment:
The new `where_close()` implementation is not iterative. I think that we previously managed to convince ourselves that multiple iterations may be necessary to resolve all close pairs, but I don't recall the detailed argument. I am trying to think of a test we could write to find out.